### PR TITLE
Add acceptance tests

### PR DESCRIPTION
**Pull request by smortex**

### CODE REVIEW

File: spec/acceptance/wifi_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+require 'spec_helper_acceptance'
+
+describe 'wifi class' do
+  it 'ensure network-manager is installed' do
+    pp = <<-MANIFEST
```
Comment (neomilium) [inline spec/acceptance/wifi_spec.rb]: Use squiggly heredoc syntax: `<<~`

File: spec/acceptance/wifi_spec.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    package { 'network-manager':
+      ensure => installed,
+    }
+    MANIFEST
+
+    apply_manifest(pp, catch_failures: true)
+  end
+
+  it 'configures Wi-Fi access points' do
+    pp = <<-MANIFEST
```
Comment (neomilium) [inline spec/acceptance/wifi_spec.rb]: Use squiggly heredoc syntax: `<<~`

**Review by neomilium — CHANGES_REQUESTED** [review]

File: spec/acceptance/wifi_spec.rb [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    MANIFEST
+
+    apply_manifest(pp, catch_failures: true)
+  end
+
+  it 'configures Wi-Fi access points' do
+    pp = <<~MANIFEST
+    # https://www.youtube.com/watch?v=bLE7zsJk4AI
+
+    wifi::infrastructure { 'Rocketjump GGGGG':
```
Comment (neomilium) [inline spec/acceptance/wifi_spec.rb]: If I run locally something like this (ie. heredoc), `rubocop` asks me to ident like this:
```ruby
puppet = <<~PUPPET
  Hello Puppet
PUPPET
```

Where these ones are bad:
```ruby
puppet = <<~PUPPET
  Hello Puppet
  PUPPET
```
```ruby
puppet = <<~PUPPET
Hello Puppet
PUPPET
```

**Review by neomilium — CHANGES_REQUESTED** [review]

File: spec/acceptance/wifi_spec.rb (hunk 3 above)
Comment (smortex) [reply spec/acceptance/wifi_spec.rb]: Not worse… Might be a good candidate to add to modulesync_config?  Needs a rubocop version bump maybe?

**Review by neomilium — APPROVED** [review]

**Pull request merged**